src/Markup/Avalonia.Markup.Xaml.Loader/AvaloniaRuntimeXamlLoader.cs

```diff
@@ -17,31 +18,15 @@ public static class AvaloniaRuntimeXamlLoader
         /// <param name="uri">The URI of the XAML being loaded.</param>
         /// <param name="designMode">Indicates whether the XAML is being loaded in design mode.</param>
         /// <returns>The loaded object.</returns>
-        public static object Load(string xaml, Assembly localAssembly = null, object rootInstance = null, Uri uri = null, bool designMode = false)
+        public static object Load(string xaml, Assembly? localAssembly = null, object? rootInstance = null, Uri? uri = null, bool designMode = false)
         {
-            Contract.Requires<ArgumentNullException>(xaml != null);
+            xaml = xaml ?? throw new ArgumentNullException(nameof(xaml));
 
             using (var stream = new MemoryStream(Encoding.UTF8.GetBytes(xaml)))
             {
                 return Load(stream, localAssembly, rootInstance, uri, designMode);
             }
         }
-        
-        /// <summary>
-        /// Loads XAML from a string.
-        /// </summary>
-        /// <param name="xaml">The string containing the XAML.</param>
-        /// <param name="configuration">Xaml loader configuration.</param>
-        /// <returns>The loaded object.</returns>
-        public static object Load(string xaml, RuntimeXamlLoaderConfiguration configuration)
```

Comment:
IIRC it was supposed to be a public API for users who want to load XAML at runtime.

Reply:
So the idea was to have configuration parameters in a separate class so we can add more of them without breaking compatibility.

Reply:
Yes, I added this configuration parameter couple of weeks ago. So no breaking changes here.
But this configuration shouldn't contain any file source related configs, like RootObject, otherwise it won't be reusable in the LoadGroup methods, where there could be multiple root objects.